massgov/mayflower-patternlab
# DP-8363 Reduce the front end performance impact of the main logo and state seal

## Description

- Break down the site logo in the header with the state seal image and the text, Mass.gov.
- For the seal, use the same file as one in the footer.
- Resize the footer seal image from 240 x 240 to 200 x 200, which is minimum required size for HiDPi/Retina display.  The file size has been reduced 5kb.

Associated Drupal PR:  [DP-8363 Reduce the front end performance impact of the main logo and state seal #2224](https://github.com/massgov/mass/pull/2224)

## Related Issue / Ticket

- [DP-8363 Reduce the front end performance impact of the main logo and state seal](https://jira.state.ma.us/browse/DP-8363)

## Steps to Test


1. Go to pages with difference display patterns with the site logo (see the screenshots below), and compare any difference between production.

<img width="1439" alt="readme2_desktop" src="https://user-images.githubusercontent.com/9633303/39437653-76d93bd8-4c6f-11e8-821c-048b8586251b.png">

<img width="1438" alt="homepage_desktop" src="https://user-images.githubusercontent.com/9633303/39437668-7fb454c2-4c6f-11e8-836a-29e4f200a387.png">

<img width="1105" alt="homepage_mobile" src="https://user-images.githubusercontent.com/9633303/39437674-853d3a76-4c6f-11e8-841e-3bcb6877ade7.png">

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
